src/main/java/org/greencodeinitiative/creedengo/java/checks/MakeNonReassignedVariablesConstants.java

```diff
@@ -103,6 +111,77 @@ public static boolean hasModifier(ModifiersTree modifiersTree, Modifier expected
         return false;
     }
 
+    private boolean hasNoLombokSetter(VariableTree variableTree) {
+        // Check if the variable is annotated with @Setter
+
+        for (AnnotationTree annotation : variableTree.modifiers().annotations()) {
+            if (annotation.annotationType().toString().equals(LOMBOK_SETTER)) {
```

Comment:
I think here you can check if this is the lombok class and not comparing string name